kotlintest-tests/kotlintest-tests-core/src/test/kotlin/com/sksamuel/kotlintest/inspectors/InspectorsTest.kt

```diff
@@ -70,6 +70,52 @@ class InspectorsTest : WordSpec() {
             "4 => java.lang.NullPointerException\n" +
             "5 => java.lang.NullPointerException"
       }
+      "failed results are truncated when failed array size is over 10" {
+        shouldThrowAny {
+          arrayOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12).forAll {
+            if (true) throw NullPointerException()
+          }
+        }.message shouldBe "0 elements passed but expected 12\n" +
+          "\n" +
+          "The following elements passed:\n" +
+          "--none--\n" +
+          "\n" +
+          "The following elements failed:\n" +
+          "1 => java.lang.NullPointerException\n" +
+          "2 => java.lang.NullPointerException\n" +
+          "3 => java.lang.NullPointerException\n" +
+          "4 => java.lang.NullPointerException\n" +
+          "5 => java.lang.NullPointerException\n" +
+          "6 => java.lang.NullPointerException\n" +
+          "7 => java.lang.NullPointerException\n" +
+          "8 => java.lang.NullPointerException\n" +
+          "9 => java.lang.NullPointerException\n" +
+          "10 => java.lang.NullPointerException\n" +
+          "... and 2 more failed elements"
+      }
+      "failed results are truncated when failed list length is over 10" {
```

Comment:
This is the exact same test as in line 73. I guess you wanted to show that the passing case also truncate message, so maybe a too quick copy/paste :P

Reply:
This code is as I intended. I intended to write test for both `Array.forAll` and `Collection.forAll` in accordance with other tests.
But, I agree with you, this test is not useful.

So, I'll write test about passing case.

Reply:
f113425